src/annotator/notebook.js

```diff
@@ -84,13 +84,17 @@ export default class Notebook extends Delegator {
     this._update();
     container.classList.add('is-open');
     container.style.display = '';
+    this.originalOverflowStyle = this.element.style.overflow;
```

Comment:
Would you mind throwing a comment in here about why we're setting `overflow:hidden` on `this.element`?

Reply:
Done.